gp-templates/translation-row.php

```diff
@@ -155,11 +155,11 @@
 							<button class="reject" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-rejected_' . $t->id ) ); ?>"><strong>&minus;</strong> <?php _e( 'Reject', 'glotpress' ); ?></button>
 							<?php endif; ?>
 							<?php if ( 'fuzzy' !== $t->translation_status ) : ?>
-							<button class="fuzzy" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-fuzzy_' . $t->id ) ); ?>"><strong>~</strong> <?php _e( 'Fuzzy', 'glotpress' ); ?></button>
+							<button class="fuzzy" tabindex="-1" data-nonce="<?php echo esc_attr( wp_create_nonce( 'update-translation-status-fuzzy_' . $t->id ) ); ?>"><strong>~</strong> <?php _e( 'Mark as fuzzy', 'glotpress' ); ?></button>
```

Comment:
There was discussion about this when the PR was done, since we don't do "Mark as Approved" it was decided to not do it for Fuzzy.